.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+    "singleQuote": true,
+    "printWidth": 120,
+    "trailingComma": "all",
+    "bracketSpacing": false
+}
```

Comment:
There rules come from app-bedrock-web

Reply:
It is a best practice to separate the refacto of prettier and the new code you want to add, here we can't see if a modification comes from the prettier refacto or not